styles/core.scss

```diff
@@ -137,3 +119,304 @@ SPDX-License-Identifier: Apache-2.0 */
     }
   }
 }
+
+.core-top-bar {
+  border-bottom: 1px solid var(--border-primary-color);
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+  padding-top: 0.75rem;
+  padding-bottom: 0.75rem;
+  width: 100%;
+  margin-bottom: 0.25rem;
+
+  > span {
+    margin-right: 1rem;
+  }
+
+  h3 {
+    color: var(--text-color-secondary);
+    font-size: 1.15rem;
+    margin: 0.25rem 0;
+    min-height: 2rem;
+  }
+
+  .right {
+    flex: 1 1 0%;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-end;
+
+    button {
+      margin: 0 0 0 1rem;
+    }
+  }
+}
+
+.core-page-row {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  display: flex;
+  flex-shrink: 0;
+  flex-flow: row wrap;
+  width: 100%;
+  padding-left: 1.25rem;
+  padding-right: 1.25rem;
+
+  @media (min-width: (v.$page-width-small-threshold + 1px)) {
+    padding-left: 2.25rem;
+    padding-right: 2.25rem;
+  }
+
+  @media (min-width: (v.$page-width-medium-threshold + 1px)) {
+    padding: 0 5rem 0 2.5rem;
+  }
+
+  @media (min-width: 1500px) {
+    padding: 0 5rem 0 2.5rem;
+  }
+
+  &.no-vertical-spacer {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  /* kill heading padding, already applied to wrapper */
+  h1,
+  h2,
+  h3,
+  h4 {
+    margin-top: 0;
+  }
+}
+
+.core-page-title {
```

Comment:
It looks like this CSS has just been copied and pasted from the staking dashboard with no real thought of how to optimise it in this repo. For example, the header tags above should probably have 0 margin by default, set in Entry, so we don't have to keep resetting it.

I think a lot of work needs to be done to this section of the PR.

Reply:
understood, I will review them all